### PR TITLE
Dba 656

### PR DESCRIPTION
**Pull request by ranbeersingh1**

### CODE REVIEW

**Review by bill-buchan — APPROVED** [review]
LGTM

File: .github/workflows/oracle-db-backup.yml [hunk 1]
```diff
@@ -135,7 +135,8 @@ jobs:
     environment: ${{needs.build_rman_target_name.outputs.TargetEnvironment}}
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/ministryofjustice/hmpps-delius-operational-automation:0.1
+      image: ghcr.io/ministryofjustice/hmpps-delius-operational-automation:0.40.0
+      options: --user root
```
[the remark this answers is not in the document]
Comment (ranbeersingh1) [reply .github/workflows/oracle-db-backup.yml]: Left over from testing, removed.

File: docker/delius-ansible-aws/Dockerfile [hunk 2]
```diff
@@ -21,7 +21,8 @@ LABEL org.opencontainers.image.authors="HMPPS Probation Webops Team (probation-w
   org.opencontainers.image.source="https://github.com/ministryofjustice/hmpps-delius-operational-automation"
 
 RUN apk update --no-cache \
-  && apk upgrade --no-cache
+    && apk upgrade --no-cache \
+    && apk add gcc musl-dev python3-dev py-setuptools build-base libffi-dev openssl-dev openssh-client git make jq bash curl
```
[the remark this answers is not in the document]
Comment (ranbeersingh1) [reply docker/delius-ansible-aws/Dockerfile]: Kept only a small number of packages

File: ansible/group_vars/dev.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 ---
 ansible_python_interpreter: /usr/bin/python3.9
-ansible_aws_ssm_bucket_name: delius-core-development-openldap-migration
+# ansible_aws_ssm_bucket_name: delius-core-development-openldap-migration
+ansible_aws_ssm_bucket_name: ldap-dev-migration20240123173113114100000001
```
[the remark this answers is not in the document]
Comment (ranbeersingh1) [reply ansible/group_vars/dev.yml:4]: Only for testing, shall change when purposed Ansible aws ssm bucket is created.

File: docker/delius-ansible-aws/Dockerfile [hunk 4]
```diff
@@ -35,4 +36,6 @@ COPY  --from=ssm-builder /go/src/github.com/session-manager-plugin/bin/linux_amd
 
 RUN adduser -D -u 1000 ansible
 
-USER ansible
+# USER ansible
+
+ENTRYPOINT /bin/bash
```
[the remark this answers is not in the document]
Comment (ranbeersingh1) [reply docker/delius-ansible-aws/Dockerfile]: Commented out.

**Pull request merged**